dotnet/src/Skills/Skills.Planning/BasicPlanner/Planner.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel.Orchestration;
+
+namespace Microsoft.SemanticKernel.Skills.Planning.BasicPlanner;
```

Comment:
Planning is a top-level concept of SK. We do not want it buried under the "Skills." namespace.

There was a "PlannerSkill" and it was leading to confusion about how to interact with the, so it was decided by the devs responsible for planner that they'd remove it earlier this week. This just reintroduces the same problem.

Reply:
Namespacing and libraries do not bury the code. We lead users with examples and documentation to show which components and features to leverage. Otherwise we would have to put everything in one folder and the code would be unmaintainable.
I'm just going to ignore the "*the responsible devs*" part, which is out of scope.

Reply:
moved to Extension package